src/game_timer.cpp

```diff
@@ -369,7 +369,6 @@ bool GameTimer::init()
 	return true;
 }
 
-//will we need this for anything else?
 CurrentTime GameTimer::get_current_time()
```

Comment:
do you know if we use this function anymore? I saw that you removed the call from `world` which didn't do anything, but I don't think that we actually use it anywhere else. I think this function was originally created because we weren't sure how we were going to use the timer, but then we ended up doing the rendering in the `game_timer` class itself, so I don't think it's necessary

Reply:
Oh nvm, I see the level manager uses it now

Reply:
Yup, using it to get the year to pass to level_manager